keps/sig-node/2238-liveness-probe-grace-period/README.md

```diff
@@ -33,16 +37,21 @@
 
 Items marked with (R) are required *prior to targeting to a milestone / release*.
 
-- [X] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
-- [X] (R) KEP approvers have approved the KEP status as `implementable`
-- [X] (R) Design details are appropriately documented
-- [X] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
-- [X] (R) Graduation criteria is in place
-- [X] (R) Production readiness review completed
-- [X] (R) Production readiness review approved
-- [X] "Implementation History" section is up-to-date for milestone
-- [X] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
-- [X] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
```

Comment:
as the text changed slightly for each of these, will review before marking as completed